core/src/main/java/bisq/core/monetary/Price.java

```diff
@@ -107,6 +109,13 @@ public long getValue() {
         return monetary.getValue();
     }
 
+    /** 
+    / @return amount of whole coins or fiat dollars as double
+     * */ 
+    public double getDoubleValue() {
```

Comment:
I don't need to use this on this PR it turns out, but it is maybe nice to have?  Let me know.